sdk/textanalytics/Azure.AI.TextAnalytics/tests/samples/Sample3_ExtractKeyPhrases.cs

```diff
@@ -16,19 +16,30 @@ public void ExtractKeyPhrases()
             string endpoint = TestEnvironment.Endpoint;
             string apiKey = TestEnvironment.ApiKey;
 
-            #region Snippet:TextAnalyticsSample3CreateClient
             var client = new TextAnalyticsClient(new Uri(endpoint), new AzureKeyCredential(apiKey));
-            #endregion
 
             #region Snippet:ExtractKeyPhrases
-            string document = "My cat might need to see a veterinarian.";
+            string document = @"My cat might need to see a veterinarian. It has been sneezing more than normal, and although my 
+                                little sister thinks it is funny, I a worried it has the cold that I got last week.
```

Comment:
nit: `I a` => `I am`. Possibly applies to other samples in which this document is used.